akka-actor/src/main/scala-2.13+/akka/util/ByteString.scala

```diff
@@ -253,6 +252,9 @@ object ByteString {
       buffer.putByteArrayUnsafe(bytes)
     }
 
+    override private[akka] def copyToArray[B >: Byte](dest: Array[B], start: Int, len: Int): Int =
```

Comment:
scalac on travis wants this to be public - I guess that makes sense because it should override a public method in the standard library?

Reply:
ah, yes, leftover from having a separate internal method